docs-site/docs/api/net/http/net.http.Destination.md

```diff
@@ -13,6 +13,10 @@ HTTP(s) destination class that holds metadata (e.g., host, port, useSSL). The de
 - Module: [http/http.js](https://github.com/SAP/xsk/tree/main/modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/http/http.js)
 - Status: `alpha`
 
+## API Changes
+Destinations in XSK make use of SAP BTP Destination service (see [xshttpdest](../../../artefacts/xshttpdest.md)).
+As such, destinations no longer live in the file tree, so the first parameter of `readDestination` representing the package is no longer needed. If provided, it will be ignored. Lookup for destinations happens based on the name.
+
```

Comment:
Please use the `!!! note` or `!!! warning` block to highlight the changes:

```
!!! note "API Changes"
    Destinations in XSK make use of SAP BTP Destination service (see [xshttpdest](../../../artefacts/xshttpdest.md)). As such, destinations no longer live in the file tree, so the first parameter of `readDestination` representing the package is no longer needed. If provided, it will be ignored. Lookup for destinations happens based on the name.
```